x/gamm/client/cli/query.go

```diff
@@ -388,6 +388,7 @@ func GetCmdSpotPrice() *cobra.Command {
 				return err
 			}
 
+			// nolint: staticcheck
```

Comment:
What's the reason for all the nolint static checks that has been added?

Reply:
it's warns on the use of deprecated structs. I think some might be true positive, can you help having a second look on which ones are?